src/Proxy/DriveItemProxy.php

```diff
@@ -287,7 +287,7 @@ public function upload($name, $content, array $options = [])
 
         $status = $response->getStatus();
 
-        if ($status != 201) {
+        if ($status != 201 && $status != 200) {
```

Comment:
`$status != 200 && $status != 201`: this order would make more sense to me :)

Reply:
sure